test/test-parse-statement.cpp

```diff
@@ -949,6 +949,33 @@ TEST_F(test_parse_statement, switch_case_without_expression) {
   }
 }
 
+TEST_F(test_parse_statement, switch_case_with_duplicated_cases) {
+  {
+    test_parser p(u8"switch (cond) {case x: case y: case y:}"_sv,
+                  capture_diags);
+    p.parse_and_visit_statement();
+    EXPECT_THAT(p.errors,
+                ElementsAre(DIAG_TYPE_2_OFFSETS(
+                    p.code, diag_duplicated_cases_in_switch_statement,
+                    first_switch_case, strlen(u8"switch (cond) {case x: case "),
+                    u8"y", duplicated_switch_case,
```

Comment:
Nit: Add a blank comment to improve code formatting:
```suggestion
                    first_switch_case, strlen(u8"switch (cond) {case x: case "),
                    u8"y", //
                    duplicated_switch_case,
```

Reply:
Fixed it  in commit https://github.com/quick-lint/quick-lint-js/pull/866/commits/0172714a51675810905d3cd0f84e72daef5d1078 and I also did a rebase to fix the merge conflicts